DuckDuckGo/AppDelegate.swift

```diff
@@ -330,7 +330,6 @@ import WebKit
 
 #if NETWORK_PROTECTION
         widgetRefreshModel.beginObservingVPNStatus()
-        AppDependencyProvider.shared.networkProtectionAccessController.refreshNetworkProtectionAccess()
```

Comment:
This line is the problem - it will deactivate the VPN when the beta active flag is disabled, but doesn't distinguish between beta or subscription users. Let's remove it completely.